# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: widget/TextEvents.h
# Commit: 3f6deedb2ffa
# Full Hash: 3f6deedb2ffae2a52c130f68cc7c0bda882bedfd
# Author: Masayuki Nakano <masayuki@d-toybox.com>
# Date: 2022-02-08 17:32:27
# Regressor Bug: 1746104
# File Overlap Count: 1
# Description:
#   Bug 1746104 - part 7: Make `eQuerySelectedText` event and `ContentEventHandler` work with no selection ranges r=m_kato
#   
#   Finally, making `WidgetQueryContentEvent::Succeeded` return `true` when there
#   is no selection when its message is `eQuerySelectedText`, and making
#   `ContentEventHandler::OnQuerySelectedText` set the no selection range data even
# ==============================================================================

diff -r 989ad9507a0c -r 3f6deedb2ffa widget/TextEvents.h
--- a/widget/TextEvents.h	Mon Feb 07 22:33:41 2022 +0000
+++ b/widget/TextEvents.h	Mon Feb 07 22:33:41 2022 +0000
@@ -1043,9 +1043,6 @@
       return false;
     }
     switch (mMessage) {
-      case eQuerySelectedText:
-        return mReply->mOffsetAndData.isSome() ||
-               mInput.mSelectionType != SelectionType::eNormal;
       case eQueryTextContent:
       case eQueryTextRect:
       case eQueryCaretRect: